src/Inkluzitron/Services/GraphPaintingService.cs

```diff
@@ -70,7 +70,7 @@ public void Dispose()
             AvatarPercentageForeground.Dispose();
         }
 
-        public async Task<Bitmap> DrawAsync(SocketGuild guild, IDictionary<string, List<QuizDoubleItem>> toplistResults)
+        public async Task<Bitmap> DrawAsync(SocketGuild guild, IDictionary<string, List<BdsmTestOrgItem>> toplistResults)
```

Comment:
ree z toho budou merge konflikty :sadge:

Reply:
To bude sranda, no